complete/_rg

```diff
@@ -319,6 +319,7 @@ _rg() {
     '(-q --quiet)'{-q,--quiet}'[suppress normal output]'
     '--regex-size-limit=[specify upper size limit of compiled regex]:regex size (bytes)'
     '*'{-u,--unrestricted}'[reduce level of "smart" searching]'
+    '--stop-on-nonmatch[stop on first non-matching line after a matching one]'
```

Comment:
Isn't this missing the `=` sign like the rest of them?

Reply:
No. The `=` sign is only used for flags that accept an argument.

Reply:
I think this shouldn't have an `=` because the option does not take any arguments. See for example the `--debug` option further up which also does not take any arguments.